src/care/DefaultMacros.h

```diff
@@ -548,6 +548,15 @@
 #define CARE_CHECKED_PARALLEL_LOOP_END(CHECK) }); \
    CARE_NEST_END(CHECK) }}
 
+#define CARE_CHECKED_STREAMED_LOOP_START(INDEX, RESOURCE, START_INDEX, END_INDEX, CHECK) { \
```

Comment:
Can you make RESOURCE the first argument of this macro (and the CARE_STREAMED_LOOP macro below)?